src/databricks/sqlalchemy/test/_extra.py

```diff
@@ -63,8 +66,39 @@ class DateTimeTZTestCustom(_DateFixture, fixtures.TablesTest):
 
     @testing.requires.datetime_implicit_bound
     def test_select_direct(self, connection):
-
         # We need to pass the TIMESTAMP type to the literal function
         # so that the value is processed correctly.
         result = connection.scalar(select(literal(self.data, TIMESTAMP)))
         eq_(result, self.data)
+
+
+class TableDDLTestCustom(fixtures.TestBase):
+    """This test confirms that a table comment can be dropped.
+    The difference with TableDDLTest is that the comment value is '' and not None after
+    being dropped.
+    """
```

Comment:
BTW this is surprising. From the [docs](https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-ddl-comment.html) we could expect the comment to be set to NULL.
But that's what I get when I test on databricks cloud.

Reply:
I adapted for this in #328 by simply coercing the `''` value into a `NoneType`. This passes SQLAlchemy's reusable tests.

Reply:
Nice 👍